### ISSUE TITLE
pptsmv2使用dml蒸馏，准确率更低了

### ISSUE THREAD

**Issue by cjplol** [PaddlePaddle/PaddleVideo#619]
我在其他参数相同的条件下，把pptsmv2的普通训练改成了dml训练（因为看官方的实验dml能涨2个点），然而准确率下降了很多，正常训练的话验证准确率最高有80多，用了dml蒸馏准确率只有不到50，想知道问题出在哪里，下面是我正常训练和dml蒸馏的配置代码：

正常训练：
MODEL: #MODEL field
    framework: "Recognizer2D" #Mandatory, indicate the type of network, associate to the 'paddlevideo/modeling/framework/' .
    backbone: #Mandatory, indicate the type of backbone, associate to the 'paddlevideo/modeling/backbones/' .
        name: "PPTSM_v2" #Mandatory, The name of backbone.
        pretrained: "data/PPTSMv2_k400_16f.pdparams" #Optional, pretrained model path.
        num_seg: 16
        class_num: 7
    head:
        name: "MoViNetHead" #Mandatory, indicate the type of head, associate to the 'paddlevideo/modeling/heads'

dml蒸馏训练：
MODEL: #MODEL field
    framework: "RecognizerDistillation" #Mandatory, indicate the type of network, associate to the 'paddlevideo/modeling/framework/' .
    freeze_params_list:
    - False
    - False
    models:
    - Teacher:
        backbone: #Mandatory, indicate the type of backbone, associate to the 'paddlevideo/modeling/backbones/' .
            name: "ResNetTweaksTSM" #Mandatory, The name of backbone.
            pretrained: "data/ResNet50_vd_ssld_v2_pretrained.pdparams" #Optional, pretrained model path.
            depth: 50 #Optional, the depth of backbone architecture.
            num_seg: 16
        head:
            name: "ppTSMHead" #Mandatory, indicate the type of head, associate to the 'paddlevideo/modeling/heads'
            num_classes: 7 #Optional, the number of classes to be classified.
            in_channels: 2048 #input channel of the extracted feature.
            drop_ratio: 0.5 #the ratio of dropout
            std: 0.01 #std value in params initialization
            num_seg: 16
    - Student:
        backbone: #Mandatory, indicate the type of backbone, associate to the 'paddlevideo/modeling/backbones/' .
            name: "PPTSM_v2" #Mandatory, The name of backbone.
            pretrained: "data/PPLCNetV2_base_ssld_pretrained.pdparams" #Optional, pretrained model path.
            num_seg: 16
            use_temporal_att: False
            class_num: 7
        head:
            name: "MoViNetHead" #Mandatory, indicate the type of head, associate to the 'paddlevideo/modeling/heads'
    loss:
        Train:
            - name: "DistillationCELoss"
              #loss_weight: 1.0  # set loss weight, default 1.0
              model_name_pairs: ["Student", "GroundTruth"]
            - name: "DistillationCELoss"
              model_name_pairs: ["Teacher", "GroundTruth"]
            - name: "DistillationDMLLoss"
              model_name_pairs: ["Student", "Teacher"]
        Val:
            - name: "DistillationCELoss"
              model_name_pairs: ["Student", "GroundTruth"]

其他地方的配置都一样，只有dml的batch_size从8调到了1（显存不够，但我觉得不至于掉30多的准确率），求大佬指点一下

**Comment by huangjun12** [comment]
降低batch_size之后，学习率有线性调整嘛
